scripts/common/prompts.sh

```diff
@@ -107,14 +79,9 @@ function prompt_for_token_ca_hash() {
     if [ -n "$KUBEADM_TOKEN_CA_HASH" ]; then
         return
     fi
-    if ! prompts_can_prompt ; then
-        bail "kubernetes.kubeadmTokenCAHash required"
```

Comment:
i do not understand why this is changing

Reply:
Same reason, we are `read -p "Kubernetes discovery token CA hash: " PROMPT_RESULT` instead.